### DISCUSSION TITLE
群晖docker配置alist反向代理关于site_url的问题

### DISCUSSION THREAD

**Discussion by Aziily** [AlistGo/alist#3239] [Q&A]
如题，在群晖上配置反向代理时，尝试将 site_url 设置为 http://localhost:5244/alist 但是无法正常访问
请问是不可以使用这种写法嘛
能否给出一个 site_url 的配置示例，感谢

**Comment by zhangbiwen** [comment]
同问

**Reply by Rektyfikowany** [reply]
site_url是从客户端上能访问的url，内网环境可以考虑`http://192.168.1.2:5244`这种方式，公网需要公网ip或者域名
